submissions/OlStaniev/DOM/script/script.js

```diff
@@ -0,0 +1,46 @@
+import episodes from './text-episode.js';
+
+const menuList = document.querySelector('.nav__list');
+const introText = document.querySelector('.introText')
+const burgerIcon = document.querySelector('.header__burgerMenu')
```

Comment:
Check your variables of naming.
Maybe the name burgerBtn will be correct. Because we have a button, not an icon.